setup.cfg

```diff
@@ -23,6 +23,7 @@ packages = find:
 install_requires =
     psycopg2
     attrs
+    pendulum
```

Comment:
add `pendulum.*` at https://github.com/peopledoc/cabbage/pull/26/files#diff-380c6a8ebbbce17d55d50ef17d3cf906R61 to please mypy.

(and maybe add type annotations in a compatible way as decribed in https://mypy.readthedocs.io/en/latest/installed_packages.html#making-pep-561-compatible-packages?)

Reply:
I updated the `setup.cfg` file.

And regarding type annotations, there is a PR (https://github.com/sdispater/pendulum/pull/320) that I need to merge at some point.